openstudio-ee.gemspec

```diff
@@ -21,15 +21,21 @@ Gem::Specification.new do |spec|
   spec.files = `git ls-files -z`.split("\x0").reject do |f|
     f.match(%r{^(test|lib.measures.*tests|spec|features)/})
   end
-  spec.bindir        = 'exe'
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir = 'exe'
+  spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '~> 2.7.0'
+  spec.required_ruby_version = '~> 3.2.2'
 
-  spec.add_dependency 'bundler', '~> 2.1'
-  spec.add_dependency 'openstudio-extension', '~> 0.7.0'
-  spec.add_dependency 'openstudio-standards', '~> 0.5.0'
+  spec.add_dependency 'bundler', '~> 2.4.10'
+  spec.add_dependency 'openstudio-extension', '~> 0.8.0'
+  spec.add_dependency 'openstudio-standards', '0.6.1'
+  spec.add_dependency 'openstudio-workflow', '~> 2.4.0'
+  spec.add_dependency 'octokit', '4.18.0' # for change logs
+  spec.add_dependency 'multipart-post', '2.4.0'
+  spec.add_dependency 'parallel', '1.19.1'
+  spec.add_dependency 'regexp_parser', '2.9.0'
+  spec.add_dependency 'addressable', '2.8.1'
```

Comment:
@wenyikuang @DavidGoldwasser i don't think we need anything in line 33-38 here...what do you think?

Reply:
```
  spec.add_dependency 'multipart-post', '2.4.0'
  spec.add_dependency 'regexp_parser', '2.9.0'
  spec.add_dependency 'addressable', '2.8.1'
```
At least for these new imported library, I didn't see any place we used them..